android/KMEA/app/src/main/java/com/keyman/engine/KMKeyboard.java

```diff
@@ -443,26 +443,42 @@ public void onDestroy() {
     dismissHelpBubble();
   }
 
+  @Override
   public void onConfigurationChanged(Configuration newConfig) {
     super.onConfigurationChanged(newConfig);
```

Comment:
What set me on this path:  we're calling methods that should be automatically called by the Android OS with parameters we're managing in order to maintain WebView and host page layout.  That... feels odd.  It _does_ seem to be the correct time to adjust Android View layout data, at least.

The question this left me with, though:  the API for Configuration suggests that it returns the "available" width and height.  Of course, this would have been computed _before_ we set the new layout parameters in place.  So... is it actually _right_ to expect the values to be current within this method?  Or, do we need to wait for layout adjustments to take place first?